toolkit/tools/pkg/imagecustomizerlib/bootcustomizer.go

```diff
@@ -206,3 +214,53 @@ func (b *BootCustomizer) WriteToFile(imageChroot *safechroot.Chroot) error {
 
 	return nil
 }
+
+func (b *BootCustomizer) ApplyChangesToGrub(varName string, newValue string) error {
```

Comment:
You do not need the `ApplyChangesToGrub` and `UpdateCmdlineValues` functions. Please delete them. The `UpdateKernelCommandLineArgs` function already does what you need.